latest-stories.html

```diff
@@ -3,63 +3,71 @@
-                <h4>
-                    PUBG Mobile, Game for Peace Global Revenue Hits $1.5 Billion in 2019
-                </h4>
-            </td>
-            <td>
-                <img src="Images/Boeing.png" height="250">
-                <small> December 18,2019
-                </small>
-                <h4>
-                    How the Boeing 737 MAX Production Halt Affects Airlines Globally
-                </h4>
-            </td>
-            <td>
-                <img src="Images/img5.jpg" height="250">
-                <small>December 18,2019
-                </small>
-                <h4>
-                    When you see something strange happening, but the software is telling you everything is fine.
-                </h4>
-            </td>
-            <td>
-                <img src="Images/cricket.jpeg" height="250">
-                <small> December 18,2019
-                </small>
-                <h4>
-                    India vs West Indies: Defending totals, Team India's Achilles heel in T20 cricket
-                </h4>
-            </td>
-            <td>
-                <img src="Images/WhatsApp_Logo_1.png" height="250">
-                <small> December 18,2019
-                </small>
-                <h4>
-                    Update Your WhatsApp If You Do Not Want a Malicious Message to Render it Useless
-                </h4>
-            </td>
-        </tr>
+    <header>
+        <h2>
+            Latest stories
+        </h2>
+    </header>
+    <!-- first article -->
+    <ul>
+        <li class="column" id="article-border-right" style="padding: 0px 10px 0px 0px;">
+            <figure>
+                <img src="images/AIRTEL.jpg" alt="">
+                <figcaption>December 18 ,2019</figcaption>
+            </figure>
+            <h4>
+                Masood Barkati, editor of children's mag Naunehal, passes away
+            </h4>
+        </li>
+        <li class="column" id="article-border-right">
+            <figure>
+                <img src="images/PubG-Game-1.jpg" alt="">
+                <figcaption>December 18 ,2019</figcaption>
+            </figure>
+            <h4>
+                1,000 primary schools shut down in KP over low enrolment
+            </h4>
+        </li>
+        <li class="column" id="article-border-right">
+            <figure>
+                <img src="images/Boeing.png" alt="">
+                <figcaption>December 18 ,2019</figcaption>
+            </figure>
+            <h4>
+                Kohli, Anushka set to marry in italy
+            </h4>
+        </li>
+        <li class="column" id="article-border-right">
+            <figure>
+                <img src="images/cricket.jpeg" alt="">
+                <figcaption>December 18 ,2019</figcaption>
+            </figure>
+            <h4>
+                Moeen urges England team to imporve off-field behaviour
+            </h4>
+        </li>
+        <li class="column" id="article-border-right">
```

Comment:
id  are unique in a document.  https://www.w3schools.com/html/html_id.asp

Reply:
you can use pseudo class selector to disable border for the last li. https://www.w3schools.com/cssref/sel_last-child.asp